pytorch_lightning/accelerators/cpu.py

```diff
@@ -10,7 +10,12 @@
 
 
 class CPUAccelerator(Accelerator):
-
+    '''
```

Comment:
can we use the " as in the rest of the docs?

Reply:
also is it correct that this docstring is attached to the class and not the method that actually raises the exception?

Reply:
Are you saying to use '' instead of `` or 
use double-quotation marks to create the docstring

Reply:
Use double quotation marks

Reply:
Understood

Reply:
> also is it correct that this docstring is attached to the class and not the method that actually raises the exception?

@awaelchli Actually I made this kind of change even in tuner, because that way the *errors* and *exceptions* show up in the documentation, else they won't. That was my logic

Reply:
It would be good to find out why it didn't show up (maybe because of the wrong quotation marks?). If the documentation is not attached to the method/function it is about, what's the point :)